### PR TITLE
More validation pipeline fixes for Irma

### PR DESCRIPTION
**Pull request by remiolsen**
This will make NouGAT incompatible with BUSCOv1. I don't think we should care about that tbh.

### CODE REVIEW

File: config_files/config_de_novo_irma.yaml [hunk 1]
```diff
@@ -44,8 +44,14 @@ Tools:
...
   options:  ["-m", "genome", "-f"]
  kmergenie:
   bin: /lupus/ngi/production/latest/KmerGenie/kmergenie
   options: [""]
+ FRC:
+  bin: /lupus/ngi/staging/latest/resources/nougat_singularity/FRC
+  options: ""
+ qaTools:
+  bin: /lupus/ngi/staging/latest/resources/nougat_singularity/qaCompute 
+  options: ""
```
Comment (remiolsen) [inline config_files/config_de_novo_irma.yaml]: @sylvinite Is this the correct path?

Comment (sylvinite) [reply config_files/config_de_novo_irma.yaml]: Use /lupus/ngi/staging/latest/resources/singularity/nougat/qaCompute instead. Makes less redundancy for methylseq pipeline and other projects

Comment (senthil10) [reply config_files/config_de_novo_irma.yaml]: Should it be `staging` ? It will work, but wouldn't `production` be more appropriate ?

Comment (remiolsen) [reply config_files/config_de_novo_irma.yaml]: Good points @sylvinite and @senthil10. How about: ```/lupus/ngi/production/latest/resources/singularity/nougat/qaCompute```

Comment (senthil10) [reply config_files/config_de_novo_irma.yaml]: Looks good to me 👍

Comment (sylvinite) [reply config_files/config_de_novo_irma.yaml]: :+1:

**Pull request merged**